src/gpu/intel/jit/ir/core.hpp

```diff
@@ -1381,9 +1381,9 @@ class int_imm_t : public expr_impl_t {
 
     size_t get_hash() const override { return ir_utils::get_hash(value); }
 
-    static expr_t shrink_type(const expr_t &e) {
```

Comment:
You can go ahead and drop this patch, it has become unnecessary in later revisions of IR GEMM.